tests/test_bmi_pro_api.py

```
@@ -8,36 +8,26 @@
-spec.loader.exec_module(app_module)
-app = app_module.app
-
 
 class TestBMIProAPI:
     """Test BMI Pro API endpoint."""
 
-    def setup_method(self):
+    def setup_method(self) -> None:
         """Set up test client."""
         os.environ["API_KEY"] = "test_key"
         self.client = TestClient(app)
```

Flake8 (7.3.0)

[error] undefined name 'app'

(F821)